db/migrate/20220722215911_change_text_profile_column_name.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeTextProfileColumnName < ActiveRecord::Migration[7.0]
+  def change
+    safety_assured {
+      rename_column :partner_profiles, :max_serve, :client_capacity
+      rename_column :partner_profiles, :program_contact_name, :primary_contact_name
+      rename_column :partner_profiles, :program_contact_phone, :primary_contact_phone
+      rename_column :partner_profiles, :program_contact_mobile, :primary_contact_mobile
+      rename_column :partner_profiles, :program_contact_email, :primary_contact_email
+    }
+  end
+end
```

Comment:
@edwinthinks @dorner I believe that use rename_column is ok, what do you think?

Reply:
FWIW,  I've been doing a fair number of these lately, because of our reworking from "diaper banks" to "essentials banks" and that's how I've handled them.

Reply:
I've been bitten in the past by renaming columns. Once it's renamed, you can't easily (for example) deploy a different branch that might not have the renaming done in the code. If you need to revert it, it's harder, etc.

It can definitely be done this way, but it is a risk. It's less risky to deploy the new columns, move all data to them,  change the code to read/write to them, validate that everything is OK, and then drop the old columns.

I guess it's up to us to decide how much risk we're comfortable with for these sorts of changes.

Reply:
For small changes, I view a single step process as less risky than a multi-step one -- fewer moving  parts means fewer ways to screw it up (for example, by not keeping the two copies in sync) -- as long as sufficient testing and validation is done on the local.     
When you're doing a massive database reorg, that's entirely a different story.

Reply:
So, we can continue with this solution @cielf @dorner @edwinthinks ?

Reply:
@italomatos I think this change is fine -- please do check that the pages changes still work & there is no left over references. If that is the case, then I'll be happy to approve and merge :)